### DISCUSSION TITLE
Add GT bonuses and Epithets to the manual

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#541] [Ideas]
Arising out of #525 .
Could the GT bonuses be briefly mentioned in the manual?
I'm not sure if the info can be added to the in-game help somewhere too.

**Comment by Ouaz** [comment]
> Could the GT bonuses be briefly mentioned in the manual?

Actually, it is in Manual 2.0 (some names and values are outdated, but all GT bonus descriptions are still valid). 
https://github.com/Keriew/augustus/blob/master/res/manual/augustus_manual_2_0.pdf (page 12 - Religion, Blessings, & Bonuses).

Manuals 2.0 and 3.1 will be merged and updated for Augustus 4.0.

**Reply by catalin-duta** [reply]
Thanks for that! 
I haven't browsed through all of the folder structure on GitHub (yet). I'm guessing it's also in the *assets* folder but I don't know where that lives on Android.

**Reply by Ouaz** [reply]
Oh, I forgot you play on Android.

Manuals 2.0 and 3.1 are shipped only with Windows version (stable), AFAIK.

On Android, there's no additional folder installed, everything is embedded into the app (augustus + assets). It only creates  .ini and .log files in the C3 folder.

Adding augustus informations to the in-game help would be possible, but there would be a lot of stuff to add (GT bonuses, an article for each new building and how they work...), and it would require to keep it carefully updated. For the moment, it's simpler to have these informations in an external PDF, I think.
